### ISSUE TITLE
`sentry-cli` hasn't been published to Docker Hub since 2.20.6

### ISSUE THREAD

**Issue by nre-ableton** [getsentry/sentry-cli#1778]
Apologies if a GitHub issue isn't the place to report this, but the `sentry-cli` Docker image hasn't been published to Docker Hub in awhile -- the last version on the [tags page](https://hub.docker.com/r/getsentry/sentry-cli/tags) is 2.20.6. Is something broken with how these images are published?

**Comment by loewenheim** [comment]
You're right, we're having some trouble with our release process. We might cut a release soon to get a new docker image published, if nothing else.

**Comment by nre-ableton** [comment]
@loewenheim Thanks for looking into this! :+1:

**Comment by HansAarneLiblik** [comment]
Seems like this is resolved! `2.23.0` is now on the hub

**Comment by nre-ableton** [comment]
Indeed, I'll close this issue as being fixed. Thanks!
